lib/get-exports.js

```diff
@@ -27,6 +27,72 @@ function getExportsForNodeBuiltIn (name) {
 
 const urlsBeingProcessed = new Set() // Guard against circular imports.
 
+/**
+ * This function looks for the package.json which contains the specifier trying to resolve.
+ * Once the package.json file has been found, we extract the file path from the specifier
+ * @param {string} specifier The specifier that is being search for inside the imports object
+ * @param {URL|string} fromUrl The url from which the search starts from
+ * @returns file to url to file export
+ */
+function resolvePackageImports (specifier, fromUrl) {
+  if (!specifier.startsWith('#')) {
+    return null
+  }
+
```

Comment:
```suggestion
```

This is already handled at the callsite.

Reply:
That is also possible (to just return `pathToFileURL(require.resolve(re, { paths: [dirname(fileURLToPath(url))] })).href` here). I am just not sure if the method name is still appropriate in that case.